File: app/src/main/java/org/evcs/android/model/user/UserCar.java
package org.evcs.android.model.user;

import androidx.annotation.NonNull;

public class UserCar {
    private int carId;
    private String year;
    public String make;
    public String model;

    public UserCar(int carId, String year) {
        this.carId = carId;
        this.year = year;
    }

    @Override
    public @NonNull String toString() {
        return (year != null ? year + " " : "") + make + " " + model + " ";
    }
}

File: app/src/main/java/org/evcs/android/model/user/UserCar.kt
package org.evcs.android.model.user

import org.evcs.android.model.Car

data class UserCar(var carId: Int, val year: String? = null) {

    var id: Int = 0
    var make: String? = null
    var model: String? = null

    override fun toString(): String {
        return (if (year != null) "$year " else "") + make + " " + model + " "
    }

    fun toCar(): Car {
        return Car(carId, make, model)
    }
}